java/kotlin/BUILD

```diff
@@ -0,0 +1,21 @@
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_jvm_library")
+
+KT_SRCS = [
```

Comment:
unless you need to reference this/filter it multiple ways, prefer to just have this list directly in the rule `srcs` field.

Reply:
I changed it a little and ended up using it in a couple different places